README.md

```diff
@@ -18,7 +18,10 @@ $ npm install express-session
 ## API
 
 ```js
-var session = require('express-session')
+const session = require('express-session')
```

Comment:
This could use the `import .. from ..` style as in expressjs/express, but I think that could be done later. However, I thought `const` is good here now.

Reply:
Sounds good to me! :)